test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/SamplesEndToEndTests_Node_MultipleProcesses.cs

```diff
@@ -19,20 +19,19 @@ namespace Microsoft.Azure.WebJobs.Script.Tests.EndToEnd
 {
     [Trait(TestTraits.Category, TestTraits.EndToEnd)]
     [Trait(TestTraits.Group, TestTraits.SamplesEndToEnd)]
-    public class SamplesEndToEndTests_Node_MultipleProcesses : IAsyncDisposable
+    public class SamplesEndToEndTests_Node_MultipleProcesses : IAsyncLifetime
     {
-        private readonly MultiplepleProcessesTestFixture _fixture;
+        private readonly MultipleProcessesTestFixture _fixture;
 
         public SamplesEndToEndTests_Node_MultipleProcesses()
         {
             // We want a new fixture for each test
-            _fixture = new MultiplepleProcessesTestFixture();
+            _fixture = new MultipleProcessesTestFixture();
```

Comment:
This is intentional to have it be an instance per test, instead of `IClassFixture<T>`?

Reply:
yeah, each of these tests are restarting the host (and worker(s)) several times. i refactored this in my last PR so that they were starting from a stable starting point rather than restarting after a bunch of timeouts and other restarts forced by other tests.